app/src/main/res/layout-land/profile_rename_activity.xml

```diff
@@ -51,22 +50,33 @@
       <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:paddingBottom="56dp"
         android:background="@drawable/general_item_background_border"
+        android:paddingBottom="56dp"
         app:layout_constraintTop_toTopOf="parent">
 
-        <org.oppia.android.app.profile.ProfileInputView
-          android:id="@+id/input_name"
+        <com.google.android.material.textfield.TextInputLayout
+          android:id="@+id/profile_rename_input"
+          style="@style/TextInputLayout"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:layout_marginStart="168dp"
           android:layout_marginTop="32dp"
           android:layout_marginEnd="168dp"
-          app:label="@string/profile_rename_label"
+          app:errorMessage="@{viewModel.nameErrorMsg}"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="parent"
-          app:layout_constraintTop_toTopOf="parent"
-          profile:error="@{viewModel.nameErrorMsg}" />
+          app:layout_constraintTop_toTopOf="parent">
+
+          <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/profile_rename_input_edit_text"
+            style="@style/TextInputEditText"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:hint="@string/profile_rename_label"
+            android:imeOptions="actionDone"
+            android:maxLength="36"
```

Comment:
how are we defining 36 here?

Reply:
@anandwana001 Actually this was missing.
for AddProfile the maxLength is 36 for the name so it must be present in Rename too.

Reply:
nice catch.